dtlstransport.go

```diff
@@ -83,6 +86,29 @@ func (t *DTLSTransport) ICETransport() *ICETransport {
 	return t.iceTransport
 }
 
+func (t *DTLSTransport) onStateChange(state DTLSTransportState) {
+	t.state = state
```

Comment:
is this missing a mutex?

Reply:
That has to be without a mutex (all the callers are holding one) will add a comment!

Reply:
fixed!